api/client/cluster/client.go

```diff
@@ -324,3 +324,11 @@ func (c *clusterClient) SetNodeConf(config *osdconfig.NodeConfig) error {
 	}
 	return nil
 }
+
+func (c *clusterClient) DelNodeConf(nodeID string) error {
```

Comment:
If delete is the opposite of `SetNode`, then it should be `Unset`. Or change `SetNodeConf` to `CreateNodeConf` and then the opposite would be `DeleteNodeConf`